web/src/Pages/UserProfile/UserProfile.tsx

```diff
@@ -320,23 +237,6 @@ const UserProfile = () => {
           </Col>
         </Row>
       </div>
-
-      <UserActionConfirmationModel
-        actionInfo={actionInfo}
-        onActionConfirmed={onDeleteProfileUserConfirmed}
-        onActionCanceled={onDeleteProfileUserCanceled}
-        openModal={openDeleteConfirmationModal}
-        errorMsg={errorMsg}
-        loading={isLoading}
-      />
-
-      <ChangePasswordModel
-        onPasswordChanged={onPasswordChangeCompleted}
-        onFieldsChanged={onFormsValueChanged}
-        onCanceled={onPasswordChangeCanceled}
-        openModal={openPasswordChangeModal}
-        errorMsg={errorMsg}
-      ></ChangePasswordModel>
```

Comment:
please check and remove the unused imports ex - UserActionConfirmationModel/  ChangePasswordModel